gui.py

```diff
@@ -305,12 +418,8 @@ def onNormalizeClip(self):
     def onExportClip(self):
         if self.last_clip and self.last_clip.audio_file:
             audio_file = self.last_clip.audio_file
-            file_name, a = (
-                QFileDialog.getSaveFileName(self,
-                                            'Export Clip : %s'
-                                            % self.last_clip.name,
-                                            expanduser('~'),
-                                            'WAVE (*.wav)'))
+            file_name, a = self.getOpenFileName(
```

Comment:
Export clip feature is no more working, it try to open a file instead of writing new one